torch_uncertainty/routines/segmentation.py

```diff
@@ -231,17 +265,59 @@ def on_test_epoch_end(self) -> None:
         self.log_dict(self.test_sbsmpl_seg_metrics.compute(), sync_dist=True)
         if isinstance(self.logger, Logger) and self.log_plots:
             self.logger.experiment.add_figure(
-                "Reliabity diagram",
+                "Calibration/Reliabity diagram",
                 self.test_sbsmpl_seg_metrics["cal/ECE"].plot()[0],
             )
             self.logger.experiment.add_figure(
-                "Risk-Coverage curve",
+                "Selective Classification/Risk-Coverage curve",
                 self.test_sbsmpl_seg_metrics["sc/AURC"].plot()[0],
             )
             self.logger.experiment.add_figure(
-                "Generalized Risk-Coverage curve",
+                "Selective Classification/Generalized Risk-Coverage curve",
                 self.test_sbsmpl_seg_metrics["sc/AUGRC"].plot()[0],
             )
+            # Plot segmentation results.
+            for i, (img, pred, tgt) in enumerate(self.sample_buffer):
```

Comment:
split into a function?

Reply:
What do you mean?

Reply:
just a separate function to keep a clear on_test_epoch_end